src/pages/docs/bootstrap-gitops-automation-with-gimlet-cli.md

```diff
@@ -116,6 +114,35 @@ First get some dummy yaml to deploy an Nginx container. Gimlet's OneChart Helm c
 ```
 helm template dummy-app -n staging onechart/onechart > manifest.yaml
 ```
+Or use the following manifest file:
```

Comment:
Can you remove the `helm template line ` completely? This tutorial should not rely on onechart knowledge.